examples/liminal-getting-started/requirements.txt

```diff
@@ -0,0 +1,2 @@
+
+pillow
```

Comment:
Remove pillow from all our examples and add something that has a compatible license